samcli/lib/init/template_modifiers/xray_tracing_template_modifier.py

```diff
@@ -2,100 +2,77 @@
 Class used to parse and update template when tracing is enabled
 """
 import logging
-from typing import List
+from typing import Any
 
 from samcli.lib.init.template_modifiers.cli_template_modifier import TemplateModifier
 
 LOG = logging.getLogger(__name__)
 
 
 class XRayTracingTemplateModifier(TemplateModifier):
+    import ruamel.yaml
+    from ruamel.yaml import YAML
+    from ruamel.yaml.comments import CommentedMap
```

Comment:
- I like that you put imports under class so that they won't be loaded once the module is imported. But this module has only this class, which means this will be imported whenever the class is used. Should we move the imports to the top into others?
- Line 13 imports entire `ruamel.yaml` but then next lines imports individual elements from `ruamel.yaml` again. Shouldn't we use one way or the other (either import everything or import only used stuff one by one)?